R/model.R

```diff
@@ -795,6 +798,7 @@ sample_method <- function(data = NULL,
                           fixed_param = FALSE,
                           validate_csv = TRUE,
                           show_messages = TRUE,
+                          chain_id = seq_len(chains),
```

Comment:
Same question as above about whether this should go at the bottom (because won't be used much) or higher up so it's next to the chain related arguments. I'm not sure, so either way is ok I guess.

Reply:
Moved up. I think that makes sense.